.gitignore

```diff
@@ -0,0 +1,26 @@
+# History files
```

Comment:
I pulled this file from [Github's gitignore repo](https://github.com/github/gitignore). It's a great resource for new projects in unfamiliar languages